MainApp/public/app/routes.js

```diff
@@ -20,6 +20,9 @@ angular.module('appRoutes', ['ngRoute']).config(function ($routeProvider, $locat
         .when('/gameconfigpage', {
             templateUrl: 'app/views/pages/admin/gameConfigPage.html' , controller: 'gameConfigCtrl', controllerAs: 'gameConf'
         })
+        .when('/manageAdmin', {
```

Comment:
Can this be `/admin/manage`?
Because
1) Every admin related URL should be prefixed with '/admin/'. This is necessary for my authentication work (PR coming up).
2) url's usually don't have camel case. makes them harder to type for the user